08-read-write-plot.qmd

```diff
@@ -123,11 +123,12 @@ fiona.supported_drivers
 Other, less common, drivers can be ["activated"](https://geopandas.org/en/stable/docs/user_guide/io.html) by manually supplementing `fiona.supported_drivers`.
 The first argument of the `geopandas` versatile data import function `gpd.read_file` is `filename`, which is typically a string, but can also be a file connection.
 The content of a string could vary between different drivers.
- In most cases, as with the ESRI Shapefile (`.shp`) or the GeoPackage format (`.gpkg`), the dsn would be a file name, such as `geodata.gpkg`.
- The driver is automatically selected based on the file extension, as demosntrated for a `.gpkg` file below:
+In most cases, as with the ESRI Shapefile (`.shp`) or the GeoPackage format (`.gpkg`), the dsn would be a file name, such as `geodata.gpkg`.
```

Comment:
Hi @michaeldorman -- I think it would be good to explain what "dsn" is (or to rephrase this sentence) -- it seems that "dsn" was not mentioned before...

Reply:
This should have been `filename`, now will correct. Thanks @Nowosad !